common/dependabot-common.gemspec

```
@@ -21,6 +21,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 2.5.0"
   spec.required_rubygems_version = ">= 2.7.3"
 
+  spec.add_dependency "activesupport", ">= 6.0.0"
```

Should we pin this to the major version or thinking the core API probably won't change so safe to allow a future v7?

```suggestion
  spec.add_dependency "activesupport", "~> 6.0.0"
```

---

Since activesupport is such a widely used library, I figured that pinning it to a major version might be annoying for other projects that want to pull in core as a dependency

---

Yeah makes sense 👍